data_model.md

```diff
@@ -0,0 +1,449 @@
+the provided data so that they can be properly paired for analysis (see
+[*Variables and Units*](#variables-and-units)). Forecasts and Observations
+each have additional required parameters described below.
+
+An **Aggregate** is an aggregation of many Observations. The Sites
+associated with each Observation identify the geographic locations a
+forecast provider may use to create a single aggregated Forecast. The
+framework may generate an aggregate Observation from the listed
+Observations.
+
+An **Organization** groups Sites, Observations, Forecasts and Users.
+Organizations provide a logical partition on which to sort objects and
+create access control rules. Organizations are not otherwise essential
+to the data model and are not described in depth here.
+
+Creation of new objects will require valid metadata for all required
+parameters. Most users will use the Solar Forecast Arbiter dashboard to
+create new objects. The API will also support creation via uploading a
+valid JSON definition of the object. Metadata for sites, forecasts, and
+observations will be returned as JSON in the Solar Forecast Arbiter API.
+
+### Site
+{: .anchor}
+
+
+Each Site will have a number of required parameters that identify the
+Site. They are:
+
+-   Name: name for the Site, e.g. Desert Rock
+
+-   Latitude: latitude of the Site in decimal degrees north of the
+    equator, e.g. 36.62373
+
+-   Longitude: longitude of the Site in decimal degrees east of the
+    prime meridian, e.g. -116.01947
+
+-   Elevation: Elevation of the Site in meters above sea level, e.g.
+    1007
+
+-   Timezone: IANA timezone of the Site, e.g.
+    Etc/GMT+8
+
+Additional parameters will be optional when creating a site:
+
+-   Network: measurement network name, e.g. SURFRAD
+
+-   Well-Known Text: describes a geometric area for a Site which may be
+    physically extended, i.e. a polygon over a city for a Site of
+    distributed generation PV systems.
+
+Sites that are solar power plants will require additional parameters
+that enable power forecasts. These parameters will be listed under a
+“modeling parameters” key as follows:
+
+-   Modeling parameters:
+
+    -   AC power: nameplate AC power rating
+
+    -   DC power: nameplate DC power rating
+
+    -   Temperature Coefficient: The temperature coefficient of DC power
+        in units of 1/C. Typically -0.002 to -0.005 per degree C.
+
+    -   Tracking type: type of tracking system, i.e. fixed, single axis,
+        two axis
+
+    -   For fixed tilt systems:
+
+        -   Surface tilt: tilt from horizontal of a fixed tilt system,
+            degrees
+
+        -   Surface azimuth: azimuth angle of a fixed tilt system,
+            degrees
+
+    -   For single-axis tracked systems:
+
+        -   Axis tilt: tilt from horizontal of the tracker axis, degrees
+
+        -   Axis azimuth: azimuth angle of the tracker axis, degrees
+
+        -   Ground coverage ratio: ground coverage ratio of a tracking
```

Comment:
ratio of total width of modules on a tracker to the distance between tracker axes.  For example, for trackers each with two modules of 1m width each, and a spacing between tracker axes of 7m, the ground coverage ratio is 2/7.

Reply:
I expect we'll require a decimal input, so maybe we should say "...the ground coverage ratio is 0.286 (=2/7)"